sw/host/tests/usbdev/usbdev_stream/stream_test.cc

```diff
@@ -0,0 +1,574 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// USB streaming data test
+//
+// Linux host-side application that receives a stream of LFSR-generated data
+// from the USB device, checks the received bytestream and then XORs it with a
+// host-side LFSR-generated byte stream to transmit back to the device.
+//
+// By default the streaming test expects a number of USB serial connections to
+// the target device, one port per endpoint:
+//
+//   /dev/ttyUSB0 - supplies and receives LFSR-generated byte stream for one/
+//                  the only endpoint
+//   /dev/ttyUSB1 - a secondary stream
+//   /dev/ttyUSB..
+//
+// Note that the mapping from device endpoints to USB port number is not
+// guaranteed, and  when multiple streams are used, it is _not_ necessarily the
+// case that ascending streams/endpoints in usbdev_stream_test are mapped to
+// a contiguous range of ascending ttyUSBi port names.
+//
+// Either or both of the initial input port and the initial output port may be
+// overridden using command line parameters.
+//
+// Usage:
+//   stream [-v<bool>][-c<bool>][-r<bool>][-s<bool>]
```

Comment:
The usage string does not contains several flags described below (`-z`, `-t`, `-v`)